src/ctapipe/utils/tests/test_astro.py

```diff
@@ -2,20 +2,65 @@
 """
 This module contains the utils.astro unit tests
 """
+import numpy as np
 from astropy import units as u
-from astropy.coordinates import SkyCoord
+from astropy.coordinates import Angle, SkyCoord
+from astropy.time import Time
 
 from ..astro import get_bright_stars
 
 
 def test_get_bright_stars():
     """
-    unit test for utils.astro.get_bright_stars(). Tests that only Zeta Tau is
-    returned close to the Crab Nebula as object brighter than mag=3.5.
+    unit test for utils.astro.get_bright_stars_with_motion().
```

Comment:
Please add a test transforming the obtained icrs coordinates into altaz.

E.g. `table_yale["ra_dec"].transform_to(AltAz(location=location, obstime=obstime))`